Description: Fix for stupid iniparser packaging
 Ubuntu for some reason packages iniparser without any .pc file
 and with the header in a subdirectory (until plucky)
Author: Constantin Piber <cp.piber@gmail.com>

Index: source/CMakeLists.txt
===================================================================
--- source.orig/CMakeLists.txt
+++ source/CMakeLists.txt
@@ -45,7 +45,6 @@ set(PRIVATE_PKG_DEPS
     pango
     cairo
     pangocairo
-    iniparser
     absl_flat_hash_map)
 
 set(ALL_PKG_DEPS ${PUBLIC_PKG_DEPS} ${PRIVATE_PKG_DEPS})
@@ -85,7 +84,7 @@ target_include_directories(
 set_target_properties(hyprtoolkit PROPERTIES VERSION ${HYPRTOOLKIT_VERSION}
                                               SOVERSION 6)
 target_link_libraries(hyprtoolkit PUBLIC OpenGL::EGL OpenGL::OpenGL
-                                         PkgConfig::deps)
+                                         PkgConfig::deps iniparser)
 
 check_include_file("sys/inotify.h" HAS_INOTIFY)
 pkg_check_modules(inotify IMPORTED_TARGET libinotify)
Index: source/src/system/Icons.cpp
===================================================================
--- source.orig/src/system/Icons.cpp
+++ source/src/system/Icons.cpp
@@ -16,7 +16,7 @@
 #include <ranges>
 
 extern "C" {
-#include "iniparser.h"
+#include "iniparser/iniparser.h"
 }
 
 using namespace Hyprtoolkit;
